src/appdistribution/client.ts

```diff
@@ -260,4 +306,41 @@ export class AppDistributionClient {
 
     utils.logSuccess(`Testers removed from group successfully`);
   }
+
+  async createReleaseTest(
+    releaseName: string,
+    devices: TestDevice[],
+    loginCredential?: LoginCredential,
+  ): Promise<ReleaseTest> {
+    try {
+      const response = await this.appDistroV1AlphaClient.request<ReleaseTest, ReleaseTest>({
+        method: "POST",
+        path: `${releaseName}/tests`,
+        body: {
+          deviceExecutions: devices.map((d) => this.mapDeviceToExecution(d)),
```

Comment:
Can simplify to:
```suggestion
          deviceExecutions: devices.map(this.mapDeviceToExecution),
```

Reply:
Done

Reply:
OK so this actually now causes a lint error because of potential `this` binding issues:

```
/Users/lkellogg/dev/firebase-tools/src/appdistribution/client.ts
  223:41  error  Avoid referencing unbound methods which may cause unintentional scoping of `this`.
If your function does not access `this`, you can annotate it with `this: void`, or consider using an arrow function instead  @typescript-eslint/unbound-method
```

What do you think, I feel like I'd rather have a verbose lambda then have to add `this: void` to the method signature...

Reply:
Oh never mind, I just moved the map function to types.ts (which I think makes sense) and now it's all good.